GOVERNANCE.md

```diff
@@ -4,83 +4,138 @@ This document defines project governance for the Strimzi project.
+### Lazy majority rules
 
-Voting about proposals is using +1, 0, -1 votes and their fractions, where:
+Lazy majority votes use +1, 0, -1 votes and their fractions, where:
 * +1 means "yes"
 * -1 means "no"
 * The numbers between +1 and -1 indicate how strongly you feel about the proposal
 
 _(Inspired by [Apache Software Foundation voting](https://www.apache.org/foundation/voting.html#expressing-votes-1-0-1-and-fractions))_
 
-Votes from maintainers are binding and count towards the approval of the proposal.
-Non-maintainers are allowed and encouraged to vote as well.
-But their votes are non-binding and do not count towards approval of the proposal.
+Binding votes are those cast by people in specific roles defined for that vote and count towards the approval of the subject of the vote.
+Non-binding may be cast by people not in any of those roles and do not count towards approval of the subject of the vote.
 
-+1 votes can be expressed by approving the proposal PR or in the comments.
-Other votes should be expressed in the comments.
-For example "+1 (binding)", or for a non-maintainer "+1 (non-binding)".
+The vote succeeds when there are least three +1 binding votes and no -1 binding votes.
+The vote should be opened for at least 3 days to give everyone enough time to vote.
+
+### Explicit majority rules
+
+An explicit majority vote is simply a +1 or -1. 
+The vote succeeds when ⅔ of cast binding votes are +1.
+The vote should be opened for at least 7 days to give everyone enough time to vote.
+
+### Votes
+
+#### Votes on changes to Governance
+
+With the exception of maintainers and component owners marking themselves as emeritus, all changes in the governance repository require an **Explicit majority** of **Maintainers**.
+
+#### Votes on adding or removing maintainers
+
+Votes to adding or removing a maintainer require an **Explicit majority** of **Maintainers**.
 
-Proposals are approved when they receive at least three +1 binding votes and no -1 binding votes.
-The proposal PR should be opened for at least 3 days to give everyone enough time to vote.
+#### Votes on adding or removing component owners
+
+Votes to add or remove a component owners require an **Explicit majority** of **Maintainers**.
+
+#### Votes on Proposals
+
+Proposals can be opened against the [proposals repository](https://github.com/strimzi/proposals).
+Proposals should cover any changes to the Strimzi project which might significantly impact its users or the project direction.
+
+Approving a proposal requires a **Lazy majority** of **Maintainers**.
+*However, non-binding votes are encouraged as a signal to maintainers of the acceptability of the proposal to the wider community.*
+
++1 votes can be expressed by approving the proposal PR or in the comments.
+Other votes should be expressed in the comments. For example "+1 (binding)", or for a non-maintainer "+1 (non-binding)".
```

Comment:
As we talk about "other votes", it might be better to have other vote examples?

```suggestion
Other votes should be expressed in the comments. For example "-1 (binding)", or for a non-maintainer "-1 (non-binding)".
```

Reply:
I reworded it to make clearer that -1 votes should come with _some_ reason. 

I think giving examples could make it overly specific, for example making some reasons for a -1 somehow more accepted (by virtue of being an example) than others. But that said, did you had a specific example in mind?